.github/workflows/demo.yml

```diff
@@ -1,13 +1,24 @@
-name: "Run Anchore Scan Action"
+name: "[Demo] Run Scan Action"
 
 on: [push, pull_request]
 
 jobs:
-  test:
+  test-image:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - uses: ./
       with:
-        image-reference: "alpine:latest"
+        image: "alpine:latest"
         debug: true
+
+  test-directory:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./
+      with:
+        path: "dir:tests/python"
```

Comment:
Is this what we'd suggest users enter (including "dir:...")? I think we may want to think through this a bit more. Is it an option to leave out the "path" variant and just ship the "image" variant, and come back to "path" when we have time to talk through it?

Reply:
That is a good catch. This is leftover from before I separated the input. I forgot to swap it out to remove the `dir:` part. I don't think this should prevent having this feature in.

Reply:
Okay, I'm good with that.